RG.CodeAnalyzer/RG.CodeAnalyzer.Test/AddMissingNamedArgumentsRefactoringTests.cs

```diff
@@ -0,0 +1,431 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using TestHelper;
+
+namespace RG.CodeAnalyzer.Test {
+	[TestClass]
+	public class AddMissingNamedArgumentsRefactoringTests : DiagnosticVerifier {
+		[TestMethod]
+		public void TestObjectCreation_AllArgumentsSupplied_DoesNotOfferRefactoring2() {
+			string test = @"
+using System;
+
+namespace ConsoleApplication1
+{
+	class Person {
+		public Person(int id, string firstName, string lastName) { }
+	}
+
+	class TypeName
+	{
+		public void Foo() {
+			var person = new Person(1, ""John"", ""Doe"");
+		}
+	}
+}";
+			Document document = CreateDocument(test, LanguageNames.CSharp);
+			
+			int position = test.IndexOf("new Person");
+			TextSpan span = new TextSpan(position, "new Person".Length);
+
+			List<CodeAction> actions = GetRefactorings(document, span);
+
+			Assert.IsFalse(actions.Any(a => a.Title.Contains("Add missing named argument")), 
+				"Should not offer refactoring when all arguments are already supplied (built-in analyzer handles this)");
+		}
```

Comment:
Duplicate test. Already covered in `TestObjectCreation_AllArgumentsSupplied_DoesNotOfferRefactoring`